jihyeon/services/userService.js

```diff
@@ -1,18 +1,24 @@
 const createUserDao = require("../models/UserDao");
 const pwValidationCheck = require("../utils/validation-check.js");
+const bcrypt = require("bcrypt");
```

Comment:
```suggestion
const bcrypt = require("bcrypt");

const createUserDao = require("../models/UserDao");
const pwValidationCheck = require("../utils/validation-check.js");
```
추가적으로 모듈을 import 하는 순서에 대한 컨벤션은 built-in 모듈 - 외부 모듈 - custom 모듈 순서입니다!